tests/test_cms_menus.py

```diff
@@ -839,6 +836,91 @@ def test_menu_with_softroot_page_rendering(self):
 
         self.assertTreeQuality(soft_root, mock_tree, 'level', 'title')
 
+    def test_menu_with_hidden_softroot_page_rendering(self):
+        """
+        Tree in fixture :
+               root
+                   aaa (soft_root)
+                       aaa1
+                           ccc
+                               ddd
+                       aaa2
+                   bbb
+        tag: show_menu 0 100 100 100
+        expected result when rendering softroot node aaa:
+         : renders just the softroot page, with no menu nodes
```

Comment:
I don't understand this, if the softroot is aaa and we render aaa then it should show the children

Reply:
Also random : here

Reply:
> I don't understand this, if the softroot is aaa and we render aaa then it should show the children

Here when we render a softroot which is hidden , it shouldnot render its children, it will render softroot and children if the node is visible